kernel/futex.c

```diff
@@ -358,20 +374,45 @@ long myst_syscall_futex(
     int val3)
 {
     long ret = 0;
+    uint32_t bitset = FUTEX_BITSET_MATCH_ANY;
+    int _op = op & ~(FUTEX_CLOCK_REALTIME | FUTEX_PRIVATE);
 
-    (void)val3;
+    if (op & FUTEX_CLOCK_REALTIME)
+    {
+        if (!((_op == FUTEX_WAIT_BITSET) ||
+              (_op == FUTEX_WAIT))) // || (op | FUTEX_WAIT_REQUEUE_PI)
```

Comment:
Suggest taking out commented out clause.

Reply:
I added that there since according to the specs `FUTEX_WAIT_REQUEUE_PI` is allowed to have the `FUTEX_CLOCK_REALTIME` set as well, in case we implement this functionality in the future. If that is more distracting than helpful, I can remove it.